hypervisor/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "hypervisor"
+version = "0.1.0"
+authors = ["Muminul Islam<muislam@microsoft.com>"]
+edition = "2018"
+
+
+[dependencies]
+anyhow = "1.0"
+arch_gen = { path = "../arch_gen" }
+thiserror = "1.0"
+libc = "0.2.69"
+devices = { path = "../devices" }
+kvm-ioctls = { git = "https://github.com/cloud-hypervisor/kvm-ioctls", branch = "ch" }
+kvm-bindings = { git = "https://github.com/cloud-hypervisor/kvm-bindings", branch = "ch", features = ["with-serde", "fam-wrappers"] }
+serde = {version = ">=1.0.27", features = ["rc"] }
+serde_derive = ">=1.0.27"
+serde_json = ">=1.0.9"
+vm-memory = { version = "0.2.0", features = ["backend-mmap", "backend-atomic"] }
+vmm-sys-util = { version = ">=0.5.0", features = ["with-serde"] }
+
+[dependencies.linux-loader]
+git = "https://github.com/rust-vmm/linux-loader"
+features = ["elf", "bzimage"]
```

Comment:
I cleaned up the cargo.toml dependencies, so that there is no cloud-hypervisor crate dependencies from this crate. See https://github.com/russell-islam/cloud-hypervisor/pull/1/commits/1d5c64bc02c10cc8d01925a8ee45ffd9ed9f9585